wlcs_anvil/src/main_loop.rs

```diff
@@ -95,23 +95,6 @@ pub fn run(channel: Channel<WlcsEvent>) {
             let mut cursor_guard = state.cursor_status.lock().unwrap();
             let mut elements: Vec<CustomRenderElements<_>> = Vec::new();
 
-            // draw input method square if any
-            let input_method = state.seat.input_method();
-            let rectangle = input_method.coordinates();
-            let position = Point::from((
-                rectangle.loc.x + rectangle.size.w,
-                rectangle.loc.y + rectangle.size.h,
-            ));
-            input_method.with_surface(|surface| {
-                elements.extend(AsRenderElements::<DummyRenderer>::render_elements(
-                    &smithay::desktop::space::SurfaceTree::from_surface(surface),
-                    &mut renderer,
-                    position.to_physical_precise_round(scale),
-                    scale,
```

Comment:
The scale was considered here. I wonder if that's why the size is wrong now?

Reply:
Thanks, that gives me some ideas about where to start debugging this.